test_template.yml

```diff
@@ -34,10 +34,29 @@ jobs:
         with:
           packages: xdot rolldice
           version: ${{ github.run_id }}-${{ github.run_attempt }}
+      - id: check-additional-pkg-dir
+        run: |
+          # xdot requires python3-numpy so check that it was installed
+          echo "::set-output name=numpy-exists::$([ -d "/usr/lib/python3/dist-packages/numpy" ] && echo "true" || echo "false")"
+          echo "::set-output name=cached-pkgs::$(ls -1 ${cache_dir}/*.tar.gz | sort)"
```

Comment:
Really this should be part of the interface which would be the output package-version-list in this case. I'll need to refactor so we expose this too and not try to test implementation.